bundler.go

```diff
@@ -321,16 +348,15 @@ func (b *Bundler) bundle(e ConfigurationEnvironment) (err error) {
 		return
 	}
 
+	fmt.Println("initial ldflags", b.ldflags.String())
 	// Build ldflags
-	var l = ldflags{
-		"s": []string{},
-		"X": []string{
-			`"main.AppName=` + b.appName + `"`,
-			`"main.BuiltAt=` + time.Now().String() + `"`,
-		},
-	}
+	b.ldflags["s"] = []string{}
```

Comment:
In fact, I think you can remove this line because I don't see what's its purpose.

Reply:
I am ok to, but the `-s` flag leaves out various debugging symbols, producing a smaller binary.  Honestly the binaries this thing creates are so big at this point that I think leaving out debugging symbols is a false economy.  I'll take it out.